ututors/index.html

```diff
@@ -43,10 +43,6 @@
     <link rel="shortlink" href="/ututors/node/995" />
 
     <title>UTutors | Home</title>
-    <link rel="stylesheet" media="all" id="fa-v4-font-face" href="ROPEEU_files/fa-v4-font-face.css">
-    <link rel="stylesheet" media="all" id="fa-v4-shims" href="ROPEEU_files/fa-v4-shims.css">
-    <link rel="stylesheet" media="all" id="fa-main" href="ROPEEU_files/fa-main.css">
-    <link rel="stylesheet" media="all" href="ROPEEU_files/css_ROJWxPVRoPnPD8h809nWBR2Y51Z69JVAy65PvU1Z0m0.css">
     <link rel="stylesheet" media="all" href="ROPEEU_files/css_Dw6MApTtXT1t7LuJrzoHYxGa3-7X3aPCWLQOW7ap6io.css">
```

Comment:
We will need to see if it is fine to delete the CSS files, for this and the other HTML files.